### PR TITLE
Request for a 6.0 nydus server

### PR DESCRIPTION
**Pull request by Quezler**
hi,

i'd like to request a bleeding edge server listing for nydus:

`<sugar-coating>`
- the official be server is offline
- i'll need a place to test stuff on
- it would benefit anyone on 6.0
- under fairly active moderation
- community server to explore on

`</sugar-coating>`

you have stated you didn't want people playing 6.0 yet, but this is just for those that somehow do.

there currently isn't anything hosted on that port yet, since i'll be awaiting a reaction on this pull before i setup an automated pipeline to keep the server up to date and such, so there's that.

**tldr** please either merge or close so i can plan my next step 🙂

### CODE REVIEW

**Comment by BlueWolfGaming** [comment]
if "under fairly active moderation" means "f***ing everything up the ass with js" then yes it is under very active moderation.

**Comment by Quezler** [comment]
(that's just regular nydus, 6.0 will be different ^)

**Comment by BlueWolfGaming** [comment]
(oh? more advanced things to touchie touchie?)

**Comment by Quezler** [comment]
my intentions with it:
- playing around with the plastanium conveyor with a larger pool of people
- porting parts of nydus 5.0 over to 6.0 (which can then be tested easily)
- discovering new unmodded client <-> server mechanics (like floors calls)
